pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+{package} {version} {:snowflake:}
+
+**Destination channel:** {Snowflake | defaults}
+
+### Links
+
+- [{ticket_number}]() 
+- [Upstream]()
+- [conda-forge]()
+- [pypi]()
+- Dependencies:
+  - ...
+
+### Changes:
+
+- ...
```

Comment:
Maybe we can add the link to the PR guidelines? (As long as it becomes/remains our source of truth)

https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/06_Team_processes/PR_contribution_procedure.md#guidelines